src/en/yakshacomics/build.gradle

```
@@ -0,0 +1,10 @@
+ext {
+    extName = 'YakshaComics'
+    extClass = '.YakshaComics'
+    themePkg = 'madara'
+    baseUrl = 'https://yakshacomics.com'
+    overrideVersionCode = 1
+    isNsfw = true
```

Do you have any examples? Otherwise:
```suggestion
    isNsfw = false
```